internal/cmd/local/command_kafka_start.go

```diff
@@ -0,0 +1,170 @@
+package local
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"runtime"
+	"strconv"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/api/types/container"
+	"github.com/docker/docker/api/types/strslice"
+	"github.com/docker/docker/client"
+	"github.com/docker/go-connections/nat"
+	specsv1 "github.com/opencontainers/image-spec/specs-go/v1"
+	"github.com/phayes/freeport"
+	"github.com/spf13/cobra"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	"github.com/confluentinc/cli/internal/pkg/errors"
+	"github.com/confluentinc/cli/internal/pkg/log"
+	"github.com/confluentinc/cli/internal/pkg/output"
+)
+
+func (c *command) newKafkaStartCommand() *cobra.Command {
+	return &cobra.Command{
+		Use:   "start",
+		Short: "Start a single-node instance of Apache Kafka.",
+		Args:  cobra.NoArgs,
+		RunE:  c.kafkaStart,
+	}
+}
+
+func (c *command) kafkaStart(cmd *cobra.Command, args []string) error {
+	dockerClient, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	if err != nil {
+		return err
+	}
+	defer dockerClient.Close()
+
+	if err := checkIsDockerRunning(dockerClient); err != nil {
+		return err
+	}
+
+	out, err := dockerClient.ImagePull(context.Background(), confluentLocalImageName, types.ImagePullOptions{})
+	if err != nil {
+		return err
+	}
+	defer out.Close()
+	_, err = io.Copy(os.Stdout, out)
+	if err != nil {
+		return err
+	}
```

Comment:
I think this should be equivalent?
```suggestion
	output.Print(string(out))
```

Reply:
will try to marshal json output